website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -86,53 +86,85 @@ HCP Terraform prioritizes and overwrites conflicting variables according to the
 
 If [prioritized](/terraform/cloud-docs/workspaces/variables#precedence-with-priority-variable-sets), variables in a global variable set have precedence over all other variables with the same key.
 
-### 2. Priority project-scoped variable sets
+### 2. Priority project-scoped, organization-owned variable sets
```

Comment:
I will admit I totally did not understand what this meant at first! I think we should really spell it out for clarity:
```suggestion
### 2. Priority organization-owned variable set scoped to a project
```